src/components/SwapWidget/ConfirmLimitOrderDrawer/index.tsx

```diff
@@ -73,67 +73,73 @@ const ConfirmLimitOrderDrawer: React.FC<Props> = (props) => {
   const inputTokenInfo = inputCurrency1?.tokenInfo;
   const outputTokenInfo = outputCurrency1?.tokenInfo;
 
-  const inputCurrency =
-    inputCurrency1 && inputCurrency1?.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : inputTokenInfo && inputTokenInfo.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : inputTokenInfo
-      ? new Token(
-          inputTokenInfo?.chainId,
-          inputTokenInfo?.address,
-          inputTokenInfo?.decimals,
-          inputTokenInfo?.symbol,
-          inputTokenInfo?.name,
-        )
-      : inputCurrency1
-      ? new Token(
-          inputCurrency1?.chainId,
-          inputCurrency1?.address,
-          inputCurrency1?.decimals,
-          inputCurrency1?.symbol,
-          inputCurrency1?.name,
-        )
-      : undefined;
+  const inputCurrency = () => {
```

Comment:
please use proper naming convention

Reply:
This name is not decided by me. I just reused it and fixed sonarlint error. 
Please have a look at diff.

Reply:
@akkikumar72 earlier it was variable name which was proper. now you have converted it into function name which is not proper function name